Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -370,7 +370,12 @@ public void addPotionMeta(final CommandSource sender, final boolean allowShortNa
             } else if (split[0].equalsIgnoreCase("duration") || (allowShortName && split[0].equalsIgnoreCase("d"))) {
                 if (NumberUtil.isInt(split[1])) {
                     validPotionDuration = true;
-                    duration = Integer.parseInt(split[1]) * 20; //Duration is in ticks by default, converted to seconds
+                    duration = Integer.parseInt(split[1]);
+                    if (stack.getType() == Material.TIPPED_ARROW) {
```

Comment:
This will (probably) cause errors in 1.15. You need to do a proper version check here

Reply:
This is not version-safe. I'd suggest adding a method to `MaterialUtil`.